aztec-up/bin/.aztec-run

```diff
@@ -91,6 +96,12 @@ while [[ "$#" -gt 0 ]]; do
     preserved_args+=("$1" "$2") # Store both argument and value
     shift 2                     # Move past argument and value
     ;;
+  --pxe.network)
+    VERSION="$2" # Set version to user-specified network (e.g. 'devnet')
+    echo "Using aztecprotocol/aztec: $VERSION"
+    preserved_args+=("$1" "$2") # Store both argument and value
+    shift 2
+    ;; # Move past argument and value
```

Comment:
Not a fan of this tbh, feels like we're trying to be too smart. I'd remove this, but have `pxe.network` read from `NETWORK` env (if it isn't already). In a future PR we could also add a check that we're running the correct version to prevent issues.